apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -1162,14 +1169,17 @@ state_channels_deposit(ChannelId, MinerPubkey) ->
 
 state_channels_withdrawal(ChannelId, MinerPubkey) ->
     MinerAddress = aec_base58c:encode(account_pubkey, MinerPubkey),
-    MinerAddress = aec_base58c:encode(account_pubkey, MinerPubkey),
+    StateHash = <<123456>>,
```

Comment:
This shows that the state hashes aren't properly checked. Please add a size check on the root hash in the transactions.